CHANGELOG.md

```diff
@@ -4,6 +4,36 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.0.1](https://github.com/voxpupuli/puppet-selinux/tree/v3.0.1) (2019-12-09)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v3.0.0...v3.0.1)
+
+**Implemented enhancements:**
```

Comment:
Usually enhancements mean a minor version bump rather than a patch.

Reply:
This seems like a bug fix: https://github.com/voxpupuli/puppet-selinux/pull/312/files/74e01a0bcd32efc72ef368a9e893b22f5e0b120f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR15

And this was a partial https://github.com/voxpupuli/puppet-selinux/pull/312/files/74e01a0bcd32efc72ef368a9e893b22f5e0b120f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR14

I guess that the second could mean a new middle bump.

Reply:
Actually, thinking about this, the Release Procedures should probably be:

1. Generate CHANGELOG and figure out what version to bump
2. Bump version in metadata.json
3. Regenerate CHANGELOG

Reply:
That's what I always do. I also use it to review the changes. See if things are (mis)labeled as well. Don't know if it's documented.

Reply:
@trevor-vaughan if you like you can update our documentation at https://github.com/voxpupuli/voxpupuli.github.io/blob/master/_docs/releasing_version.md (that's the source for https://voxpupuli.org/docs/releasing_version/)